src/handlers/HeyHandler.ts

```diff
@@ -1,21 +1,35 @@
 import { singleton, inject } from "tsyringe";
-
+import { Either } from "fp-ts/lib/Either";
 import Handler from "./Handler";
 import HandlerConfig from "./HandlerConfig";
 import { GroupMeInfo } from "../services/GroupMeService";
-import { Either } from "fp-ts/lib/Either";
 import MessagingService from "../services/MessagingService";
 
 @singleton()
-export default class HeyHandler implements Handler {
-  private static readonly regexps = [new RegExp("hey loki", "i")];
+export default class HeyHandler extends Handler<GroupMeInfo> {
   readonly config: HandlerConfig = {
-    regexps: HeyHandler.regexps,
+    regexp: new RegExp("hey loki", "i"),
   };
 
-  constructor(@inject("MessagingService") private readonly messagingService: MessagingService) {}
+  constructor(@inject("MessagingService") private readonly messagingService: MessagingService) {
+    super();
+  }
+
+  /**
+   * Returns a boolean saying whether or not the handler should handle the information is is given.
+   * @param groupMeInfo
+   */
+  shouldHandle = (groupMeInfo: GroupMeInfo): boolean => {
+    const passed = this.config.regexp.test(groupMeInfo.text);
+    return passed;
```

Comment:
nit
```suggestion
    return this.config.regexp.test(groupMeInfo.text);
```

Reply:
I had it this way because conceivably there will be future checks like user who sent it, etc. Doesn't matter to me tho.